src/core/ext/filters/client_channel/lb_policy/child_policy_handler.h

```diff
@@ -14,6 +14,13 @@
 // limitations under the License.
 //
 
+#include <grpc/support/port_platform.h>
```

Comment:
Shouldn't these includes be inside of the `#ifndef GRPC_CORE_EXT_FILTERS_CLIENT_CHANNEL_LB_POLICY_CHILD_POLICY_HANDLER_H` block?

Why doesn't the sanity check catch this?

Reply:
yeah that got borked... i guess it doesn't catch it because we haven't implemented such a check

Reply:
(fixed this instance)

Reply:
We do have a check that the `#ifndef` guard is present in header files.  Can we make that check enforce that any includes are inside that guard?

Reply:
It's certainly possible, I'm just not offering to pick that work up at this point.